.github/workflows/depviz.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: Doozers/depviz-action@e68e52e8761f640c9dd3bb3dd5859c7f4d29bbe5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          targets: 'moul/depviz-test'
+          gen-json: 'true'
```

Comment:
Missing the part that commits the generated files on a specified branch

Reply:
you can overwrite the default `branch: args` but it exist a default one.